.github/workflows/auto-build.yml

```diff
@@ -0,0 +1,249 @@
+name: Auto build image
+on:
+  issue_comment:
+    types:
+      - created
+jobs:
+  issue_comment:
+    name: Auto build image
+    if: ${{ github.event.comment.body == '/buildv1.20.14' }}
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+
+      - name: Check outputs
+        run: |
+          BUILD=${{github.event.comment.body}}
+          echo ${BUILD#*d}
```

Comment:
![image](https://user-images.githubusercontent.com/69408038/150265228-1d710bbf-9233-4401-8e3f-30b206a5595f.png)
you can split the string like this to get the version.